packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/saml2assert/use-flow.md

```diff
@@ -38,3 +38,43 @@ Note the parameters that are being passed:
     "id_token": "eyJraWQ.....h7BYbgCzQ"
 }
 ```
+
+### Base64-encode the client ID and client secret
+
+Use this section to Base64 encode the client ID and secret. When you finish encoding, you can then use the encoded client ID and secret in the HTTP Authorization header in the following format: `'authorization: Basic <Base64 encoded client ID and secret>'`
+
+**If you are using macOS or Linux**:
+
+1. Sign in to your Okta organization with your administrator account.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Select the application that you want to use, and then on the **General** tab, copy the **Client ID** and **Client secret**.
+1. Launch your preferred text editor and then paste the client ID and secret into a new file.
+1. Place the client ID and secret on the same line and insert a colon between them: `clientid:clientsecret`
+1. Copy the `clientid:clientsecret` line to the clipboard.
+1. Launch a terminal and enter the following command, replacing `clientid:clientsecret` with the value that you just copied.
+
+    `echo -n clientid:clientsecret | base64`
+
+1. Copy the value that is returned.
+
+> **Note:** If the value that is returned is broken into more than one line, return to your text editor and make sure that the entire results are on a single line with no text wrapping.
+
+**If you are using Windows**:
+
+1. Sign in to your Okta organization with your administrator account.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Select the application that you want to use, and then on the **General** tab, copy the **Client ID** and **Client secret**.
+1. Launch your preferred text editor and then paste the client ID and secret into a new file.
+1. Save the file to `C:\temp` and name the file `appCreds.txt`.
+1. In Windows Explorer, right-click `C:\temp`, and then select **CMD Prompt Here** from the context menu.
+1. Enter the following command to encode the client ID and client secret:
+
+    `copycertutil -encode appCreds.txt appbase64Creds.txt`
+
+1. Locate and open `appbase64Creds.txt` in `C:\temp`, copy its contents, and then close the file.
+
+> **Note:** Delete the `appCreds.txt` and the `appbase64Creds.txt` files after you finish.
+
+### Validate access token
```

Comment:
I think "Validate access tokens" is already mentioned in the nextsteps.md snippet?

Reply:
Yup, removed.